src/macros.rs

```diff
@@ -0,0 +1,316 @@
+//! Macro's for easy initialization of date and time values.
+
+/// Create a `NaiveDate` with a statically known value.
```

Comment:
Link the `NaiveDate`.

Since the macro does not have a return type the docs will not automatically link to `NaiveDate`.

So change to a link

     /// Create a [`NaiveDate`] with ...

at the bottom, add the link target

    /// [`NaiveDate`]: crate::naive::NaiveDate

This nitpick goes for the rest of the PR.

If you use VS Code, it will highlight the link to turquoise blue when it is correctly set.

<img width="357" alt="Screenshot 2024-01-30 102402" src="https://github.com/chronotope/chrono/assets/815261/c9865ade-2d55-4779-8931-25b31f4c6f60">